website/docs/d/org_group.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_group"
+sidebar_current: "docs-vcd-datasource-org-group"
+description: |-
+Provides a data source for VMware Cloud Director Organization groups.
+---
+
+# vcd\_org\_group
+
+Provides a data source for VMware Cloud Director Organization groups. This can be used to fetch organization groups already defined in `SAML` or `LDAP`.
```

Comment:
For easier eye-scanning:

```suggestion
Provides a data source for VMware Cloud Director Organization Groups. This can be used to fetch organization groups already defined in `SAML` or `LDAP`.
```

Reply:
Changed, thanks!